api/hasura/cron/epochs.ts

```diff
@@ -283,6 +328,22 @@ export async function notifyEpochStart({
         organization_id: epoch.circle.organization.id,
       });
     }
+
+    const membersData = (epoch.circle?.users || [])
+      .map(u => ({
```

Comment:
why do we map this into an object with `email: me@me.com` and then filter it to `data.email` couldn't we just map it to an array of the raw emails on the first iteration and not use the filter at all?

Reply:
@crabsinger 
the filter is to discard users without emails from the notifications.
could you give an example of your approach?

Reply:
@sshmm like so https://github.com/coordinape/coordinape/pull/2339/commits/2b123137b51096a18385834a64b904f7edacf055

reduce can do both operations in a single iteration vs iterating over the collection twice, which may be a bit more compelx code wise, but faster with large sets.